src/openforms/forms/api/serializers/form.py

```diff
@@ -220,6 +261,9 @@ class FormSerializer(PublicFieldsSerializerMixin, serializers.ModelSerializer):
     translations = ModelTranslationsSerializer()
 
     registration_backends = FormRegistrationBackendSerializer(many=True, required=False)
+    authentication_backends_configuration = FormAuthenticationBackendSerializer(
```

Comment:
let's go with a shorter name:

```suggestion
    authentication_backends = FormAuthenticationBackendSerializer(
```